x-pack/plugins/fleet/public/applications/fleet/sections/epm/screens/detail/index.tsx

```diff
@@ -198,13 +241,16 @@ export function Detail() {
               { isDivider: true },
               {
                 content: (
+                  // eslint-disable-next-line @elastic/eui/href-or-on-click
                   <EuiButton
                     fill
                     isDisabled={!hasWriteCapabilites}
                     iconType="plusInCircle"
                     href={getHref('add_integration_to_policy', {
                       pkgkey,
                     })}
+                    onClick={handleAddIntegrationPolicyClick}
```

Comment:
I don't know how this is handled internally, but will focusing this button (say, with <kbd>Tab</kbd>) and hitting <kbd>Enter</kbd> trigger everything correctly?

Reply:
Yeah, it all should work as "normal" - including right-click the link/button and selecting "open in a new window"

Reply:
verified manually. synthetic events indeed